src/lib/statistics/chunk_statistics/counting_quotient_filter.cpp

```diff
@@ -0,0 +1,163 @@
+#include "counting_quotient_filter.hpp"
+
+#include <cmath>
+#include <iostream>
+#include <string>
+
+#include "utils/murmur_hash.hpp"
+#include "resolve_type.hpp"
+#include "storage/storage_manager.hpp"
+#include "storage/table.hpp"
+#include "storage/create_iterable_from_column.hpp"
+#include "types.hpp"
+
+using gqf2::qf_init;

```

Comment:
Can't you just do `using namespace qpf2` instead of importing each function individually?

Reply:
Yes but lint does not like it.

Reply:
Is there an option for lint exceptions?

Reply:
`using namespace lol;  // NOLINT`
lint is pest